packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -327,7 +327,7 @@ export class ConnectionManager implements IConnectionManager {
         });
     }
 
-    public dispose(error?: ICriticalContainerError, switchToReadonly?: boolean) {
+    public dispose(error?: ICriticalContainerError, switchToReadonly: boolean = true) {
```

Comment:
Should this default to false to keep same behavior as previously in the PR?  If so... can you write a test to highlight the bug and prove the fix?

Reply:
Before this PR, disposing `ConnectionManager` would switch to readonly mode